example/Makefile

```diff
@@ -1,26 +1,32 @@
-.PHONY: default run build clean
```

Comment:
styleguyide suggested putting all phony directives above the target rather than collected all in one place